app/controllers/admin/users_controller.rb

```diff
@@ -1,9 +1,16 @@
 module Admin
   class UsersController < ApplicationController
+    using Decoration
```

Comment:
@LucasAnderson1984  FYI, I don't think you need to do this anymore... draper works with AR query results as well as Arrays now.